/trace/group/mcgaughey/hariharr/miniconda3/envs/mace/lib/python3.9/site-packages/e3nn/o3/_wigner.py:10: FutureWarning: You are using `torch.load` with `weights_only=False` (the current default value), which uses the default pickle module implicitly. It is possible to construct malicious pickle data which will execute arbitrary code during unpickling (See https://github.com/pytorch/pytorch/blob/main/SECURITY.md#untrusted-models for more details). In a future release, the default value for `weights_only` will be flipped to `True`. This limits the functions that could be executed during unpickling. Arbitrary objects will no longer be allowed to be loaded via this mode unless they are explicitly allowlisted by the user via `torch.serialization.add_safe_globals`. We recommend you start setting `weights_only=True` for any use case where you don't have full control of the loaded file. Please open an issue on GitHub for any issues related to this experimental feature.
  _Jd, _W3j_flat, _W3j_indices = torch.load(os.path.join(os.path.dirname(__file__), 'constants.pt'))
from run_train args: Namespace(config=None, name='mha-lr-mace-PP-vama-5A', seed=123, work_dir='.', log_dir='./logs', model_dir=None, checkpoints_dir=None, results_dir=None, downloads_dir=None, device='cpu', default_dtype='float64', distributed=False, log_level='INFO', error_table='PerAtomMAE', model='MACE', r_max=5.0, radial_type='bessel', num_radial_basis=8, num_cutoff_basis=5, pair_repulsion=False, distance_transform='None', interaction='RealAgnosticResidualInteractionBlock', interaction_first='RealAgnosticResidualInteractionBlock', max_ell=3, correlation=1, num_interactions=2, MLP_irreps='16x0e', radial_MLP='[64, 64, 64]', hidden_irreps=None, num_channels=256, max_L=2, gate='silu', scaling='rms_forces_scaling', avg_num_neighbors=1, compute_avg_num_neighbors=True, compute_stress=False, compute_forces=True, train_file='./custom_dataset/dimer_datasets/vama_updated_dimer_PP_5_train.xyz', valid_file='./custom_dataset/dimer_datasets/vama_updated_dimer_PP_5_test.xyz', valid_fraction=0.1, test_file='./custom_dataset/dimer_datasets/vama_updated_dimer_PP_5_test.xyz', test_dir=None, multi_processed_test=False, num_workers=0, pin_memory=True, atomic_numbers=None, mean=None, std=None, statistics_file=None, E0s='average', foundation_filter_elements=True, heads=None, multiheads_finetuning=True, weight_pt_head=1.0, num_samples_pt=1000, subselect_pt='random', pt_train_file=None, pt_valid_file=None, keep_isolated_atoms=False, energy_key='energy', forces_key='forces', virials_key='REF_virials', stress_key='REF_stress', dipole_key='REF_dipole', charges_key='REF_charges', loss='weighted', forces_weight=1000.0, swa_forces_weight=10.0, energy_weight=10.0, swa_energy_weight=1000.0, virials_weight=1.0, swa_virials_weight=10.0, stress_weight=1.0, swa_stress_weight=10.0, dipole_weight=1.0, swa_dipole_weight=1.0, config_type_weights='{"Default":1.0}', huber_delta=0.01, optimizer='adam', beta=0.9, batch_size=2, valid_batch_size=2, lr=0.01, swa_lr=0.001, weight_decay=5e-07, amsgrad=True, scheduler='ReduceLROnPlateau', lr_factor=0.8, scheduler_patience=5, lr_scheduler_gamma=0.9993, swa=True, start_swa=200, ema=True, ema_decay=0.99, max_num_epochs=300, patience=5, foundation_model=None, foundation_model_readout=True, eval_interval=3, keep_checkpoints=False, save_all_checkpoints=False, restart_latest=False, save_cpu=False, clip_grad=10.0, wandb=False, wandb_dir=None, wandb_project='', wandb_entity='', wandb_name='', wandb_log_hypers=['num_channels', 'max_L', 'correlation', 'lr', 'swa_lr', 'weight_decay', 'batch_size', 'max_num_epochs', 'start_swa', 'energy_weight', 'forces_weight'])
args model: MACE
2024-12-30 08:03:18.475 INFO: ===========VERIFYING SETTINGS===========
2024-12-30 08:03:18.475 INFO: MACE version: 0.3.7
2024-12-30 08:03:18.475 INFO: Using CPU
2024-12-30 08:03:18.593 INFO: ===========LOADING INPUT DATA===========
2024-12-30 08:03:18.593 INFO: Using heads: ['default']
2024-12-30 08:03:18.593 INFO: =============    Processing head default     ===========
Traceback (most recent call last):
  File "/trace/group/mcgaughey/hariharr/mace_exploration/fourier_attention_mace/mace_mha/./mace/cli/run_train.py", line 783, in <module>
    main()
  File "/trace/group/mcgaughey/hariharr/mace_exploration/fourier_attention_mace/mace_mha/./mace/cli/run_train.py", line 64, in main
    run(args)
  File "/trace/group/mcgaughey/hariharr/mace_exploration/fourier_attention_mace/mace_mha/./mace/cli/run_train.py", line 201, in run
    collections, atomic_energies_dict = get_dataset_from_xyz(
  File "/trace/group/mcgaughey/hariharr/mace_exploration/fourier_attention_mace/mace_mha/mace/tools/scripts_utils.py", line 53, in get_dataset_from_xyz
    atomic_energies_dict, all_train_configs = data.load_from_xyz(
  File "/trace/group/mcgaughey/hariharr/mace_exploration/fourier_attention_mace/mace_mha/mace/data/utils.py", line 227, in load_from_xyz
    atoms_list = ase.io.read(file_path, index=":")
  File "/trace/group/mcgaughey/hariharr/miniconda3/envs/mace/lib/python3.9/site-packages/ase/io/formats.py", line 793, in read
    format = format or filetype(filename, read=isinstance(filename, str))
  File "/trace/group/mcgaughey/hariharr/miniconda3/envs/mace/lib/python3.9/site-packages/ase/io/formats.py", line 965, in filetype
    fd = open_with_compression(filename, 'rb')
  File "/trace/group/mcgaughey/hariharr/miniconda3/envs/mace/lib/python3.9/site-packages/ase/io/formats.py", line 597, in open_with_compression
    return open(filename, mode)
FileNotFoundError: [Errno 2] No such file or directory: './custom_dataset/dimer_datasets/vama_updated_dimer_PP_5_train.xyz'
